lib/hemp/orm/sql_helper.rb

```diff
@@ -0,0 +1,14 @@
+require "fileutils"
+
+module Hemp
+  module Orm
+    module SqlHelper
+      FileUtils.mkdir_p "db"
+      @@db = SQLite3::Database.new File.join "db", "development.sqlite3"
+
+      def self.execute(*sql_args)
+        @@db.execute *sql_args
```

Comment:
Ambiguous splat operator. Parenthesize the method arguments if it's surely a splat operator, or add a whitespace to the right of the `*` if it should be a multiplication.